diff --git a/chrome/browser/prefs/BUILD.gn b/chrome/browser/prefs/BUILD.gn
index 79f7e90a7b9c4db0dcc08d3c4ce365f05bde3a7f..4e08da06283eb43ae3bc8368c487683f3c728fe9 100644
--- a/chrome/browser/prefs/BUILD.gn
+++ b/chrome/browser/prefs/BUILD.gn
@@ -207,6 +207,7 @@ source_set("impl") {
     "//services/preferences/tracked:features",
     "//ui/webui:buildflags",
   ]
+  import("//brave/browser/sources.gni") deps += brave_chrome_browser_prefs_impl_deps
 
   if (enable_print_preview) {
     deps += [
