diff --git a/chrome/browser/extensions/BUILD.gn b/chrome/browser/extensions/BUILD.gn
index ccc575f181a303b63339c865bdb8dc68f94b1124..a2d97a3cec58788909331568361cc38b478178ac 100644
--- a/chrome/browser/extensions/BUILD.gn
+++ b/chrome/browser/extensions/BUILD.gn
@@ -2,6 +2,7 @@
 # Use of this source code is governed by a BSD-style license that can be
 # found in the LICENSE file.
 
+import("//brave/browser/extensions/resources.gni")
 import("//build/config/chromebox_for_meetings/buildflags.gni")
 import("//build/config/features.gni")
 import("//build/config/ozone.gni")
@@ -641,6 +642,7 @@ source_set("extensions") {
     "//ui/native_window_tracker",
     "//ui/shell_dialogs",
   ]
+  deps += brave_extensions_resources
   public_deps = [
     "//chrome/browser/extensions/api/messaging",
     "//chrome/browser/media/webrtc",
