diff --git a/chrome/browser/ui/browser_window/internal/BUILD.gn b/chrome/browser/ui/browser_window/internal/BUILD.gn
index b410f257a76d631ca6bbaac1d2ef67d1fa3ed7c5..be00fad41006432376b38657cb61f7d49cff2310 100644
--- a/chrome/browser/ui/browser_window/internal/BUILD.gn
+++ b/chrome/browser/ui/browser_window/internal/BUILD.gn
@@ -206,6 +206,7 @@ source_set("internal") {
       "//chrome/browser/ui:browser_select_file_dialog_controller",
     ]
   }
+  import("//brave/browser/ui/browser_window/internal/sources.gni") deps += brave_browser_window_internal_deps
 }
 
 source_set("unit_tests") {
