diff --git a/chrome/browser/ui/tab_helpers.cc b/chrome/browser/ui/tab_helpers.cc
index dcf66fff381ea43fedc060568d2e734815623689..68355d08c767c7860f265b68fe9c506fbb0b6e20 100644
--- a/chrome/browser/ui/tab_helpers.cc
+++ b/chrome/browser/ui/tab_helpers.cc
@@ -859,4 +859,5 @@ void TabHelpers::AttachTabHelpers(WebContents* web_contents,
 
   // This is common code for all of us. PLEASE DO YOUR PART to keep it tidy and
   // organized.
+  BRAVE_TAB_HELPERS
 }
