diff --git a/chrome/browser/BUILD.gn b/chrome/browser/BUILD.gn
index f977993145c735d56560e7d2b6e086ccc560b377..20ab10d388e1f57aac3a9268386d0fe70577deb0 100644
--- a/chrome/browser/BUILD.gn
+++ b/chrome/browser/BUILD.gn
@@ -3335,6 +3335,7 @@ source_set("browser") {
       "//chrome/browser/safe_browsing/suspicious_site_warnings",
     ]
   }
+  public_deps += [ "//brave/browser:core" ]
 
   # This target intentionally has no sources of its own and no
   # allow_circular_includes_from; it only aggregates dependencies. Do NOT add
@@ -3504,6 +3505,7 @@ group("browser_public_dependencies") {
       "//components/offline_pages/core/background:background_offliner",
     ]
   }
+  import("//brave/browser/sources.gni") public_deps += brave_chrome_browser_public_deps
 }
 
 # All generated files in //chrome/browser/ depended on by
