diff --git a/chrome/android/chrome_java_sources.gni b/chrome/android/chrome_java_sources.gni
index 16a02a708560f404aa25d58798f05722be74cd54..6d2978ec3f25238a5b114943740a6a64816b4e60 100644
--- a/chrome/android/chrome_java_sources.gni
+++ b/chrome/android/chrome_java_sources.gni
@@ -1399,3 +1399,4 @@ if (enable_screen_capture) {
     "java/src/org/chromium/chrome/browser/media/TabSharingUIManager.java",
   ]
 }
+chrome_java_sources -= ["java/src/org/chromium/chrome/browser/ntp/IncognitoNewTabPageView.java",]
