diff --git a/chrome/browser/about_flags.cc b/chrome/browser/about_flags.cc
index 4a57f8e22fcec1836f2250275343d6e9256ec2b7..6b7e4439e830d4a6ffa4a042d1bfcde56de64474 100644
--- a/chrome/browser/about_flags.cc
+++ b/chrome/browser/about_flags.cc
@@ -4998,6 +4998,7 @@ constexpr char kEnterpriseIsolatedModeInternalName[] =
 
 // When adding a new choice, add it to the end of the list.
 const FeatureEntry kFeatureEntries[] = {
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
 // Include generated flags for flag unexpiry; see //docs/flag_expiry.md and
 // //tools/flags/generate_unexpire_flags.py.
 #include "build/chromeos_buildflags.h"
