diff --git a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
index 96a7957e309abf6cd62bbbaef69b8f9ca5b8258c..b7bb1330dacfcc84d7e276c51990286c6bc8e079 100644
--- a/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
+++ b/chrome/browser/ui/views/download/bubble/download_toolbar_ui_controller.cc
@@ -723,6 +723,7 @@ void DownloadToolbarUIController::UpdateIcon() {
   GetImageBadge(browser_view_)
       ->UpdateImage(is_active, progress_download_count, progress_color,
                     badge_background_color);
+  UpdateIcon_BraveImpl(browser_view_, *new_icon, state_, active_, action_item_);
 
   // Do not show the progress ring when there is no in progress download.
   if (state_ == IconState::kComplete || progress_info_.download_count == 0) {
