diff --git a/chrome/browser/ui/tabs/public/tab_features.h b/chrome/browser/ui/tabs/public/tab_features.h
index 3e25579ed8a372a53134c0c5ef17798f85c99ba5..2a362b7f8f4de7a9c4cbbd9d21f29b748d02242f 100644
--- a/chrome/browser/ui/tabs/public/tab_features.h
+++ b/chrome/browser/ui/tabs/public/tab_features.h
@@ -240,7 +240,7 @@ class TabInterface;
 class TabFeatures {
  public:
   TabFeatures();
-  ~TabFeatures();
+  virtual ~TabFeatures();
 
   TabFeatures(const TabFeatures&) = delete;
   TabFeatures& operator=(const TabFeatures&) = delete;
@@ -387,12 +387,13 @@ class TabFeatures {
   }
 
   // Called exactly once to initialize features.
-  void Init(TabInterface& tab, Profile* profile);
+  virtual void Init(TabInterface& tab, Profile* profile);
 
   static ui::UserDataFactoryWithOwner<TabInterface>&
   GetUserDataFactoryForTesting();
 
  private:
+  friend class BraveTabFeatures;
   bool initialized_ = false;
 
   // Returns the factory used to create owned components.
