diff --git a/components/autofill/core/common/autofill_features.h b/components/autofill/core/common/autofill_features.h
index 7694cde2b9cd37a27d8399f0fe8ea5633070f9c0..b51c6da4b2785cd7ae2a5b41a82f35a3ce6382ab 100644
--- a/components/autofill/core/common/autofill_features.h
+++ b/components/autofill/core/common/autofill_features.h
@@ -111,7 +111,7 @@ COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE_PARAM(std::string,
                            kAutofillAiSavePromptSurveyDeclinedTriggerId);
 COMPONENT_EXPORT(AUTOFILL)
-DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiServerModel);
+BASE_DECLARE_FEATURE(kAutofillAiServerModel);
 COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE_PARAM(base::TimeDelta, kAutofillAiServerModelCacheAge);
 COMPONENT_EXPORT(AUTOFILL)
@@ -165,7 +165,7 @@ COMPONENT_EXPORT(AUTOFILL)
 DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(
     kAutofillAiWalletVehicleRegistration);
 COMPONENT_EXPORT(AUTOFILL)
-DECLARE_FEATURE_WITH_MOBILE_COUNTRY_RESTRICTION(kAutofillAiWithDataSchema);
+BASE_DECLARE_FEATURE(kAutofillAiWithDataSchema);
 COMPONENT_EXPORT(AUTOFILL)
 BASE_DECLARE_FEATURE(kAutofillAllowFillingModifiedInitialValues);
 COMPONENT_EXPORT(AUTOFILL)
