diff --git a/content/browser/BUILD.gn b/content/browser/BUILD.gn
index 76a97af0f62e4bce4987b3ea91b14265e69eaaa9..6908179779927e67786babd5edf8eadd6f128a68 100644
--- a/content/browser/BUILD.gn
+++ b/content/browser/BUILD.gn
@@ -3616,6 +3616,7 @@ source_set("browser") {
     deps += [ "//components/vrp_flags" ]
     public_deps += [ "//components/vrp_flags:mojom" ]
   }
+  import("//brave/content/browser/sources.gni") sources += brave_content_browser_sources deps += brave_content_browser_deps
 }
 
 if (is_android) {
