diff --git a/DEPS b/DEPS
index f2c90747907e203ba586801b9ba13c0e47e62b99..6bfd8876770db3ee001478bfc0151fa5530e4bf5 100644
--- a/DEPS
+++ b/DEPS
@@ -4524,7 +4524,7 @@ hooks = [
     # Update the Mac toolchain if necessary.
     'name': 'mac_toolchain',
     'pattern': '.',
-    'condition': 'checkout_mac or checkout_ios',
+    'condition': 'False',
     'action': ['python3', 'src/build/mac_toolchain.py'],
   },
   {
