diff --git a/ios/chrome/browser/flags/about_flags.mm b/ios/chrome/browser/flags/about_flags.mm
index f183552ede04682434fd2e28ad0aea76420fbbaa..efedf7f8607b45340a914a8dc6deb2472d6deb93 100644
--- a/ios/chrome/browser/flags/about_flags.mm
+++ b/ios/chrome/browser/flags/about_flags.mm
@@ -1471,6 +1471,7 @@ const FeatureEntry::FeatureVariation
 //
 // When adding a new choice, add it to the end of the list.
 constexpr auto kFeatureEntries = std::to_array<flags_ui::FeatureEntry>({
+    BRAVE_ABOUT_FLAGS_FEATURE_ENTRIES
     {"build-external-privacy-context",
      flag_descriptions::kBuildExternalPrivacyContextName,
      flag_descriptions::kBuildExternalPrivacyContextDescription,
