diff --git a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
index 22b1af43b3db8aa99607eac9fd4067b8853dc6b8..38b72ab7997ada63c87d8dc868cd8807e73df57b 100644
--- a/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
+++ b/ios/chrome/browser/shared/model/prefs/browser_prefs.mm
@@ -505,6 +505,7 @@ void RegisterLocalStatePrefs(PrefRegistrySimple* registry) {
   registry->RegisterBooleanPref(kMetricsConsentRestructureFeatureState, false);
   registry->RegisterTimePref(kObsoleteManagementPlatformLastLogTime,
                              base::Time());
+  brave::RegisterLocalStatePrefs(registry);
 }
 
 void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
@@ -997,6 +998,7 @@ void RegisterProfilePrefs(user_prefs::PrefRegistrySyncable* registry) {
   registry->RegisterBooleanPref("autofill.wallet_import_enabled", true);
   registry->RegisterBooleanPref("sync.autofill_wallet_import_enabled_migrated",
                                 false);
+  brave::RegisterBrowserStatePrefs(registry);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1025,6 +1027,7 @@ void MigrateObsoleteLocalStatePrefs(PrefService* prefs) {
 
   // Added 07/2026.
   prefs->ClearPref(kObsoleteManagementPlatformLastLogTime);
+  brave::MigrateObsoleteLocalStatePrefs(prefs);
 }
 
 // This method should be periodically pruned of year+ old migrations.
@@ -1100,6 +1103,7 @@ void MigrateObsoleteProfilePrefs(PrefService* prefs) {
   // Added 08/2026.
   prefs->ClearPref("autofill.wallet_import_enabled");
   prefs->ClearPref("sync.autofill_wallet_import_enabled_migrated");
+  brave::MigrateObsoleteProfilePrefs(prefs);
 }
 
 void MigrateObsoleteUserDefault() {
