diff --git a/third_party/blink/renderer/platform/runtime_enabled_features.json5 b/third_party/blink/renderer/platform/runtime_enabled_features.json5
index be6a489f1cb8b1dc09db8110a252b1e77d63e015..30cff45b4cc19857b6bb07057b82d20d38d3c115 100644
--- a/third_party/blink/renderer/platform/runtime_enabled_features.json5
+++ b/third_party/blink/renderer/platform/runtime_enabled_features.json5
@@ -297,6 +297,7 @@
     {
       // Interest Group JS API/runtimeflag.
       name: "AdInterestGroupAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       origin_trial_feature_name: "AdInterestGroupAPI",
       implied_by: ["Fledge", "Parakeet"],
@@ -407,6 +408,7 @@
     },
     {
       name: "AIPromptAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: {
         "Win": "stable",
@@ -434,6 +436,7 @@
     },
     {
       name: "AIPromptAPIMultimodalInput",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -488,7 +491,7 @@
       origin_trial_feature_name: "AIProofreaderAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -503,7 +506,7 @@
       origin_trial_feature_name: "AIRewriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -512,6 +515,7 @@
     },
     {
       name: "AISummarizationAPI",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: {
         "Win": "stable",
         "Mac": "stable",
@@ -539,7 +543,7 @@
       origin_trial_feature_name: "AIWriterAPI",
       origin_trial_os: ["win", "mac", "linux", "chromeos"],
       origin_trial_allows_third_party: true,
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -1300,7 +1304,7 @@
       // objects that live on the compositor thread.
       // [1] https://drafts.csswg.org/css-animations-2/#timeline-triggers
       name: "CompositorTimelineTrigger",
-      status: "stable",
+      status: "experimental", // Brave: revert when crbug.com/541924666 is fixed.
     },
     {
       name: "CompressionDictionaryTransport",
@@ -1376,6 +1380,7 @@
       // https://github.com/WICG/controlled-frame/blob/main/README.md for more
       // info.
       name: "ControlledFrame",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       public: true,
       status: "stable",
     },
@@ -2935,6 +2940,7 @@
     {
       // In-development features for the File System Access API.
       name: "FileSystemAccessAPIExperimental",
+      base_feature: "none",
       status: "experimental",
     },
     {
@@ -2949,6 +2955,7 @@
     {
       // Non-OPFS File System Access API.
       name: "FileSystemAccessLocal",
+      base_feature: "none",
       status: {"default": "stable"},
     },
     {
@@ -3071,6 +3078,7 @@
     },
     {
       name: "Fledge",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -3786,7 +3794,7 @@
         "Android": "test",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -4144,6 +4152,7 @@
     // "experimental" is the support on other platforms.
     {
       name: "MiddleClickAutoscroll",
+      base_feature: "none",
       status: "test",
     },
     // Killswitch for crbug.com/40062462 fix.
@@ -4615,6 +4624,7 @@
     {
       // PARAKEET ad serving runtime flag/JS API.
       name: "Parakeet",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       origin_trial_feature_name: "Parakeet",
     },
     {
@@ -4787,6 +4797,7 @@
       //
       // It also has some feature params defined throughout the codebase.
       name: "Prerender2",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -5012,6 +5023,7 @@
     // but we still keep this flag for future phase tests.
     {
       name: "ReduceUserAgentMinorVersion",
+      base_feature_status: "enabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
     },
     {
@@ -6432,7 +6444,7 @@
         "ChromeOS": "stable",
         "default": "",
       },
-      base_feature_status: "enabled",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       copied_from_base_feature_if: "overridden",
     },
     {
@@ -6607,6 +6619,7 @@
     },
     {
       name: "UserMediaElement",
+      base_feature_status: "disabled",  // feature state is enforced via plaster rewrite.
       status: "stable",
       public: true,
     },
@@ -7405,5 +7418,10 @@
       // crbug.com/435623334.
       name: "XSLTSpecialTrial",
     },
+    {
+      name: "BraveIsInTorContext",
+      public: true,
+      base_feature: "none",
+    },
   ],
 }
