diff --git a/config/gni/devtools_grd_files.gni b/config/gni/devtools_grd_files.gni
index 6f89c723578cc500aeadb69244031c203f45b93d..b1ef8f7ce8c892e823210544f4b1177a573aece1 100644
--- a/config/gni/devtools_grd_files.gni
+++ b/config/gni/devtools_grd_files.gni
@@ -2807,6 +2807,7 @@ grd_files_unbundled_sources = [
   "front_end/ui/visual_logging/LoggingState.js",
   "front_end/ui/visual_logging/NonDomState.js",
 ]
+import("//brave/third_party/devtools-frontend/src/config/gni/sources.gni") grd_files_unbundled_sources += brave_grd_files_unbundled_sources
 
 # Confirm that there is no overlap between the two lists.
 _grd_files_bundled_sources =
